Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -161,6 +161,11 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
     public func isSubfeatureEnabled(_ subfeature: any PrivacySubfeature,
                                     versionProvider: AppVersionProvider,
                                     randomizer: (Range<Double>) -> Double) -> Bool {
+        // TODO: Do not ship this - temporary, only for testing
+        if let networkProtectionSubfeature = subfeature as? NetworkProtectionSubfeature, networkProtectionSubfeature == NetworkProtectionSubfeature.waitlist {
+            return true
```

Comment:
We could remove this, and I can add checks into my waitlist branch to force the feature on where appropriate. That way we can merge this without affecting anything.